src/agent/include/cmd_ln_parser.hpp

```diff
@@ -13,10 +14,15 @@ class CommandlineParser
     /// @brief Constructor
     /// @param argc The number of command line arguments
     /// @param argv The command line arguments
-    CommandlineParser(int argc, char** argv)
+    CommandlineParser(int argc, char** argv, const std::vector<std::string> options)
```

Comment:
Missing new parameter in the description.

Reply:
Done!